README.md

```diff
@@ -16,3 +16,53 @@ As a user, I would like to render my feature server side only to improve page pe
 
 ## What's next?
 One of the many features of PageBuilder is the ability to allow non-technical users to customize the appearance or functionality of their pages without requiring a developer. In the next step, we'll learn how we, as developers, can make such functionality available in the PageBuilder UI based on a set of options. Switch to branch `fusion-06` to see the completed code for this next step.
+
+If you are continuing this training from the last user story, then keep working on your current branch USERNAME-fusion-training.
+
+If you are starting this training from the sixth user story (task Fusion-06), then follow the steps under "What's next?" on the README file in the (https://github.com/wapopartners/Fusion-Training-User-Stories/blob/fusion-01/README.md)[fusion-01] branch - but checkout from fusion-05 instead.
+
+## New Task: User Story 06
+As a user, I would like to customize my feature in the PageBuilder UI based on a set of options.
+
+## Guide
+Building from our Color Block feature, we're going to add the ability to set the background color of our div in PageBuilder. However, you can add any customization you'd like
+
+1. From root, change directory into the folder of your feature (from our previous example in fusion-03, that will be `components/features/sample/color-block/`) and open the deafult.jsx file of your component
+
+2. Add in a custom field that will allow PageBuilder editors to input a hex code. To do this, you'll need to update the propTypes of your component to include custom fields like so:
```

Comment:
I wonder if we should add in a bit more context about how this works since it's likely a pattern they haven't encountered before? Maybe just a link to docs on custom fields? https://redirector.arcpublishing.com/alc/arc-products/pagebuilder/fusion/documentation/recipes/adding-custom-fields.md?version=2.6

Reply:
Yes, I like the idea of referencing ALC here (with redirector of course). Also lets not use the version query parameter for whenever fusion releases a new one we dont have to update these docs.

Reply:
great idea, I've added a link to the docs 👍